packages/flutter_tools/lib/src/asset.dart

```diff
@@ -147,16 +148,21 @@ class _ManifestAssetBundle implements AssetBundle {
     }
 
     final String assetBasePath = globals.fs.path.dirname(globals.fs.path.absolute(manifestPath));
-
-    final PackageMap packageMap = PackageMap(packagesPath, fileSystem: globals.fs);
+    final PackageConfig packageConfig = await loadPackageConfigUri(
+      globals.fs.file(packagesPath).absolute.uri,
```

Comment:
This needs to be absolute until we unpin and upgrade to https://pub.dev/packages/package_config#193